src/role-helpers.js

```diff
@@ -186,7 +186,7 @@ function getRoles(container, {hidden = false} = {}) {
     }, {})
 }
 
-function prettyRoles(dom, {hidden, includeDescription}) {
+function prettyRoles(dom, {hidden, includeDescription} = {}) {
```

Comment:
Is this change necessary as part of this commit? :)

Reply:
When invoking prettyRoles in tests without this change,
I'm getting the following error:

<img width="1688" alt="image" src="https://github.com/testing-library/dom-testing-library/assets/6171622/73f22f38-01fe-47f9-b640-d2598ff025e5">

Reply:
Because this is just for internal usage I don't think this change is needed.

Reply:
done.